### ISSUE TITLE
alarm inactive when phone sleeps

### ISSUE THREAD

**Issue by francogrex** [ataradov/alarm-me#6]
very nice and simple app. However I suppose because it has no "service", the alarm does not go off if the phone's CPU goes into "deep sleep" (e.g. after about 5 min with screen off). Can that be fixed?

**Comment by ataradov** [comment]
Yes, I noticed that sometimes alarms go off for me with 1-10 minute delay. 
I'm not sure what do to about that. I'm not an Android developer and I don't have a whole lot of time to troubleshoot this right now.

If you have some instructive article or piece of code I can look at, I might do something about. If it takes me days to figure out, then I've got other projects that are much more fun than working with stupid Android.
